packages/react-native/ReactAndroid/src/test/java/com/facebook/react/bridge/BaseJavaModuleTest.kt

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package com.facebook.react.bridge
+
+import com.facebook.react.turbomodule.core.interfaces.TurboModule
+import com.facebook.testutils.shadows.ShadowSoLoader
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when` as whenever
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+
+/** Tests for [BaseJavaModule] and [JavaModuleWrapper] */
+@Config(shadows = [ShadowSoLoader::class])
+@RunWith(RobolectricTestRunner::class)
+class BaseJavaModuleTest {
+  private lateinit var methods: List<JavaModuleWrapper.MethodDescriptor>
+  private lateinit var moduleWrapper: JavaModuleWrapper
+  private lateinit var generatedMethods: List<JavaModuleWrapper.MethodDescriptor>
+  private lateinit var generatedModuleWrapper: JavaModuleWrapper
+  private lateinit var arguments: ReadableNativeArray
+
+  @Before
+  fun setup() {
+    val moduleHolder = ModuleHolder(MethodsModule())
+    moduleWrapper = JavaModuleWrapper(null, moduleHolder)
+    methods = moduleWrapper.methodDescriptors
+    val generatedModuleHolder = ModuleHolder(GeneratedMethodsModule())
+    generatedModuleWrapper = JavaModuleWrapper(null, generatedModuleHolder)
+    generatedMethods = generatedModuleWrapper.methodDescriptors
+    arguments = mock(ReadableNativeArray::class.java)
+  }
+
+  private fun findMethod(mname: String, methods: List<JavaModuleWrapper.MethodDescriptor>?): Int {
+    var posn = -1
+    for (i in methods!!.indices) {
+      val md = methods[i]
+      if (md.name === mname) {
+        posn = i
+        break
+      }
+    }
+    return posn
+  }
+
+  @Test(expected = NativeArgumentsParseException::class)
+  @Throws(Exception::class)
```

Comment:
This should be removed.

Reply:
Thank you for your reviews, I would love to know why this should be removed

Reply:
The `@Test(expected = NativeArgumentsParseException::class)` bit should be enough to fulfill the purpose of this test, the `@Throws(Exception::class)` part doesn't add anything there in terms of information, neither it's required for proper compilation.